<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLogin</class>
 <widget class="QDialog" name="DialogLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>设备:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DevType">
         <item>
          <property name="text">
           <string>USB_CAN_2E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>USB_CAN_E</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>设备号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CanIndex">
         <item>
          <property name="text">
           <string>CAN0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CAN1</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>波特率:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="BoardRate">
         <item>
          <property name="text">
           <string>250Kb/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>125Kb/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500Kb/s</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>端口号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="PortIndex">
         <item>
          <property name="text">
           <string>Port0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port1</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>89</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>89</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
